src/utilities.jl

```diff
@@ -38,9 +38,19 @@ function is_lower_snake_case_alphanumeric(x::AbstractString, also_allow=())
            all(i -> i in ALPHANUMERIC_SNAKE_CASE_CHARACTERS || i in also_allow, x)
 end
 
-####
-#### zstd_compress/zstd_decompress
-####
+#####
+##### arrrrr i'm a pirate
```

Comment:
Why are the extensions here necessary? Could add a comment to that effect. It'd be nice to upstream the definitions to TimeSpans to avoid the piracy.

Reply:
> It'd be nice to upstream the definitions to TimeSpans to avoid the piracy.

This definitely isn't the responsibility of TimeSpans, though - this all Onda-specific stuff.

Reply:
Why is it needed on the Onda side?

Reply:
To (de)serialize TimeSpans to the Arrow schema specified by the Onda format